.circleci/Makefile

```diff
@@ -1,12 +1,12 @@
 # Builder version
-CI_VERSION ?= go1.11-k8s1.10.4-helm2.7.2-minikube0.25
-CI_HUB ?= istio
+CI_VERSION ?= 2019-04-22
```

Comment:
does anyone know where the source is that is used to build this container?  We need to move forward from helm 2.7.2 - its long since deprecated.

Reply:
@sdake line 17 of .circleci/Dockerfile

Minikube also has a lot of updates since 0.25.